config/thinking_sphinx.yml.example

```diff
@@ -1,103 +0,0 @@
-common: &common 
-  charset_table: "&, 0..9, a..z, _, A..Z->a..z, U+00C0->a, U+00C1->a, \
```

Comment:
mozno blba otazka, ale kde sa nacita toto? kedze si to odtialto vyhodil, ale nevidim to nikde.